NOTE-contributing.md

```diff
@@ -0,0 +1,11 @@
+Contributions to this repository are intended to become part of Notes governed by the [Software and Document License](https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document). By committing here, you agree to that licensing of your contributions.
+
+If you are not the sole contributor to a contribution (pull request), please identify all contributors in the pull request comment.
```

Comment:
do we need this? the repo manager doesn't do anything with non-rec track documents at the moment, so no tool will be consuming that information

Reply:
makes sense, indeed - then this LGTM :)